code/modules/admin/admin_fax_panel.dm

```diff
@@ -126,6 +122,8 @@
 			if(stamp)
 				fax_paper.add_stamp(stamp_class, params["stampX"], params["stampY"], params["stampR"], stamp)
 			
+			fax_paper.update_static_data(usr) // OK, it's work, and update UI. 
```

Comment:
:+1:

Reply:
> 👍

??

Reply:
This is an acceptable fix

Reply:
So... end of review?

Reply:
Probably we're in a code freeze right now though so can't merge